src/app/components/LabeledInput/LabeledInput.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import LabeledInput from "./LabeledInput";
+
+export default {
+  title: "Component/LabeledInput",
+  component: "LabeledInput",
+};
+
+export const Email = (): JSX.Element => (
+  <LabeledInput
+    label="E-Mail"
+    id="mail"
+    htmlFor="mail"
```

Comment:
I think these are always the same? Maybe it's enough to just add `id`?

Reply:
I first cascaded it, but then it was harder to style it in CSS.
If I just add id, are they still connected?

Reply:
You can say `htmlFor={id}` to connect it :)